src/redux/missions/mission.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+const FETCH_MISSIONS = 'FETCH_MISSIONS';
+const TOGGLE_MISSION = 'TOGGLE_MISSION';
+const missions = [];
+const url = 'https://api.spacexdata.com/v3/missions';
+
+export const FetchMissionsHandler = () => async (dispatch) => {
+  const response = await axios.get(url);
+  return dispatch({ type: FETCH_MISSIONS, payload: response.data });
+};
+
+export const ToggleMission = (id) => ({ type: TOGGLE_MISSION, id });
+
+const missionsReducer = (state = missions, action) => {
+  switch (action.type) {
+    case FETCH_MISSIONS:
```

Comment:
- [ ] Please store just the required data from the API. It's a way to reduce the volume of area needed to store the data and makes your app run lightly.

<img width="546" alt="Screen Shot 2022-11-19 at 2 38 10 PM" src="https://user-images.githubusercontent.com/47673311/202864248-6987d3a3-075f-4600-8c9c-6bc0c0cf88fe.png">